jobs/refresh-hiv-data.yml

```diff
@@ -24,25 +22,11 @@ configuration:
               datasource: "warehouse-hiv.yml"
               table: "${tableName}"
               schema: "sql/schemas/${tableName}.sql"
-        iterations:
-          - tableName: "all_lab_orders"
-          - tableName: "all_lab_results"
-          - tableName: "hiv_dispensing"
-          - tableName: "hiv_patient"
-          - tableName: "hiv_patient_program"
-          - tableName: "hiv_regimens"
-          - tableName: "hiv_status"
-          - tableName: "hiv_tests"
-          - tableName: "hiv_viral_load"
-          - tableName: "hiv_visit"
-          - tableName: "ovc_program_encounters"
-          - tableName: "pmtct_infant_delivery"
-          - tableName: "pmtct_contacts"
-          - tableName: "pmtct_visits"
-          - tableName: "pmtct_labs"
-          - tableName: "pmtct_pregnancy"
-          - tableName: "tb_lab_results"
-          - tableName: "tb_screening"
```

Comment:
Do we do TB screening outside of hiv forms?  I thought this was an HIV thing, but defer to you guys.